PRFCS/prfc-1.md

```diff
@@ -51,50 +55,50 @@ Returns information about the Token implemented by this contract.
 fn allowance(owner: PublicAddress, spender: PublicAddress) -> u64
 ```
 
-Returns the number of tokens currently in the spender's allowance that they can spend on the ‘owner’s’ behalf.
+Returns the amount of tokens currently in the spender's allowance that they can spend on the owner’s behalf.
```

Comment:
Is this better?
`Returns the amount of token that the spender can currently spend on behalf of the owner.`